apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -101,6 +106,15 @@ export function TokenDetails({ coinType }: TokenDetailsProps) {
         refetchInterval: DELEGATED_STAKES_QUERY_REFETCH_INTERVAL,
     });
 
+    const { data: supplyIncreaseVestingObjects } = useGetAllOwnedObjects(
```

Comment:
we don't need to fetch all the objects in that package, as long as we find one, that's enough to show the hint